rdkit/Chem/Descriptors.py

```diff
@@ -181,6 +181,21 @@ def MinAbsPartialCharge(mol, force=False):
 MinAbsPartialCharge.version = "1.0.0"
 
 from rdkit.Chem.EState.EState import MaxEStateIndex, MinEStateIndex, MaxAbsEStateIndex, MinAbsEStateIndex
+from rdkit.Chem.QED import qed
+
+
+def _FingerprintDensity(mol, func, *args, **kwargs):
```

Comment:
This differences between the bit vector and count vector form of this function are bothering me.
Here's the asymptotic behavior for the the two different densities for linear carbon chains with between 1 and 100 atoms:
![image](https://cloud.githubusercontent.com/assets/540511/26771893/6c9e8c00-49c2-11e7-9d7a-6612996a888f.png)
and
![image](https://cloud.githubusercontent.com/assets/540511/26771904/779bc7da-49c2-11e7-901c-d97735cbf4b3.png)

I'm not sure which is "more correct", but the fact that they are so different is disturbing.